### PR TITLE
Compatibility with WP 6.4

### PR DESCRIPTION
**Pull request by olafleur-godaddy**
### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Comment by AnthonyLedesma** [comment]
RichText multiline prop was deprecated. We seem to be using multiline props in a few places. Might need refactors here.

**Review by olafleur-godaddy — COMMENTED** [review]
Nice job!

File: .github/workflows/test-e2e.yml [hunk 1]
```diff
@@ -6,51 +6,42 @@ on:
...
-  #   uses: ./.github/workflows/test-e2e-cypress.yml
-  #   with:
-  #     wpVersion: "WordPress/WordPress#6.1"
-  #   concurrency:
-  #     group: chrome-defaults-major-before-previous-major
-  #     cancel-in-progress: true
-
-  chrome_e2e_previous_major:
-    name: Chrome with previous major WP version
+  chrome_e2e_gammut_tests:
```
[the remark this answers is not in the document]
Comment (AnthonyLedesma) [reply .github/workflows/test-e2e.yml:9]: 😬 I don't think Gammut, means much of anything. 😬 
Gamut however does have meaning. Ill fix that.

```
gam·ut
/ˈɡamət/
noun
1.
the complete range or scope of something.
"the whole gamut of human emotion"
```

File: .github/workflows/test-e2e-cypress.yml [hunk 2]
```diff
@@ -92,11 +96,12 @@ jobs:
           plugins: '["."]'
           themes: '["https://downloads.wordpress.org/theme/go.zip"]'
 
+
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version-file: '.nvmrc'
-          cache: 'yarn'
+            node-version-file: '.nvmrc'
```
[the remark this answers is not in the document]
Comment (AnthonyLedesma) [reply .github/workflows/test-e2e-cypress.yml:103]: Does seem to be working btw. 🤷

**Review by AnthonyLedesma — APPROVED** [review]
A complete test matrix has been established and is passing with these modifications. 

https://github.com/godaddy-wordpress/coblocks/actions/runs/6709822699

**Comment by AnthonyLedesma** [comment]
I found a quick CSS fix, that I will be pushing up without letting the tests all run again.

**Pull request merged**